packages/squiggle-lang/src/js/cli/utils.ts

```diff
@@ -0,0 +1,71 @@
+import { SqProject } from "../SqProject";
+import { SqValueTag } from "../SqValue";
+
+export const measure = (cb: () => void, times = 1) => {
```

Comment:
maybe something like ``measureTime`` or ``timer``?

Reply:
I think there should be two distinct modes:
1. Get the results of 1 result. Maybe show the time by default or don't. Flag to show bindings or not.
2. Run N times and get time, plus maybe % errors or similar.

Reply:
I think we should generally try to have performance measurement done within the language. For example, it's very useful to measure part of a program, or compare two ways of writing the same thing, checking that they're the same and showing both timings. It's easier, and more consistent across interfaces, to do these with some sort of utility function.

As for what the CLI should do, I would lean towards just being able to output time for 1 run. It's not so obvious what an average of multiple runs is measuring, with compilation time and JS's JIT and who knows what else contributing.